UI.py

```diff
@@ -0,0 +1,109 @@
+from getpass import getpass
+# importing 'mysql.connector' as mysql 
+import mysql.connector as mysql
+
+class DbConnectionError(Exception):
+    pass
+
+#DB Connection
+def _connect_to_db():
+    cnx = mysql.connect(
+        host = "localhost",
+        user = "root",
+        passwd = "password",    #Reviewer Password
+        database = "walk2zero"
+    )
+    return cnx
+
+#Function to accept User Input
+def user_reg():
+    first_name = input("Please enter your first name = ")
+    last_name = input("Please enter your last name = ")
+    email = input("Please enter your email address = ")
+    password = getpass("Please enter a desired password = ")
+    new_registration(first_name,last_name,email,password)
+
+#Function to log-in an exisiting user
+def login():
+    check_email = input("Please enter your email address = ")
+    check_password = getpass("Please enter your password = ")
+    verify_login(check_email,check_password)
+
+
```

Comment:
I think we'll have to create something that makes a user choose whether to login or sign up but this can come later

Reply:
The menu() function sorts it out. We can add more choices to it as the project builds up :)